torch/_inductor/fx_passes/fuse_attention.py

```diff
@@ -165,7 +165,7 @@ def _sfdp_params_check(match):
         attn_mask = attn_mask_node.meta["val"]
         if (
             not isinstance(attn_mask, torch.Tensor)
-            or (attn_mask.dtype != query.dtype and attn_mask.dtype != torch.bool)
+            or attn_mask.dtype != query.dtype
```

Comment:
Update the comment at https://github.com/pytorch/pytorch/pull/100619/files/a177cfadac8b75e97dc7bd412f9509adb0c30f88..8cf3142ec663db6109796e6a22ead27c5f6283d8#diff-d2f5fd73265c51de38cecb3b0309a2e555e8260feb7bd16a963fd9b89321a537L162 too?